Makefile

```diff
@@ -736,6 +736,7 @@ git-gaia-node-modules: gaia_node_modules.revision
 npm-cache:
 	@echo "Using pre-deployed cache."
 	npm rebuild marionette-js-runner
```

Comment:
we need to do this so that the python environment is setup as expected because with our suite, we don't use mozharness (which normally sets up the python things)